packages/devextreme/js/__internal/ui/overlay/overlay.ts

```diff
@@ -1257,7 +1291,10 @@ class Overlay<
   _moveToContainer(): void {
     const $wrapperContainer = this._positionController.$container;
 
-    this._$wrapper.appendTo($wrapperContainer);
+    if ($wrapperContainer !== undefined) {
```

Comment:
Should we check exactly for undefined here? Can be replaced by just?
```suggestion
    if ($wrapperContainer) {
```

Reply:
> Should we check exactly for undefined here?

Yes